app/code/core/Mage/Core/Helper/Data.php

```diff
@@ -774,7 +774,10 @@ public function mergeFiles(
                 } else {
                     $targetMtime = filemtime($targetFile);
                     foreach ($srcFiles as $file) {
-                        if (!file_exists($file) || @filemtime($file) > $targetMtime) {
+                        if (!file_exists($file)) {
+                            // no translation intentionally
+                            throw new Exception(sprintf('File %s not found.', dirname($file)));
```

Comment:
If I delete the file `js/prototype/window.js`, in the log file I get the an incomplete path: `File /usr/local/var/www/magento/js/prototype not found` because `dirname` returns the parent dir's name, so there's no need to use it:
```suggestion
                            throw new Exception(sprintf('File %s not found.', $file));
```